src/MoBi.Engine/Sbml/UnitDefinitionImporter.cs

```diff
@@ -131,7 +140,7 @@ public IDimension ConvertUnit(UnitDefinition unitDefinition)
          if (dimension != Constants.Dimension.NO_DIMENSION)
          {
             _sbmlInformation.MobiDimension[sbmlUnit] = dimension;
-            _unitConvertionDictionary.Add(sbmlUnit, new UnitConvertionInfo() { Dimension = dimension, Unit = dimension.DefaultUnit, Rate = 1 });
+            _unitConversionDictionary.Add(sbmlUnit, new UnitConversionInfo {Dimension = dimension, Unit = dimension.BaseUnit, Rate = 1});
```

Comment:
I am pretty sure you mean BaseUnit and not DefaultUnit. We always need to convert to BaseUnit.